themes/nav-community-v1/layouts/partials/header.html

```diff
@@ -1,23 +1,53 @@
-<!-- main menu area -->
-<header class="stickey">
-   <nav class="main-menu">
-      <div class="container">
-         <!-- Brand and toggle get grouped for better mobile display -->
-         <div class="navbar-header">
-            <button type="button" class="navbar-toggle collapsed custom-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><i class="fa fa-bars fa-lg"></i>
-            </button>
-            <div class="logo-brand">
-               <a href="/">
-                  <img src="{{ "images/nav-logo.png" | relURL }}" alt="logo">
-               </a>
+<div id="wrapper">
+
+   <div id="header-wrapper">
+      <header class="stickey">
+         <nav class="main-menu">
+            <div class="col-sm-9">
+               <div class="logo-brand">
+                  <a href="/">
+                     <img src="{{ "images/nav-logo.png" | relURL }}" alt="logo">
+                  </a>
+               </div>
+            </div>
+            <div class="col-sm-3">
+               <button type="button" class="hamburger is-closed" data-toggle="offcanvas">
+                  <span class="hamb-top"></span>
+                  <span class="hamb-middle"></span>
+                  <span class="hamb-bottom"></span>
+               </button>
             </div>
-         </div>
-         <!-- Collect the nav links, forms, and other content for toggling -->
-         <div class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1">
-            {{ template "theme/partials/header_menu.html" . }}
-         </div>
-         <!-- /.navbar-collapse -->
-      </div>
-      <!-- /.container-fluid -->
-   </nav>
-</header>
+         </nav>
+      </header>
+   </div>
+
+    <nav class="navbar navbar-fixed-top" id="sidebar-wrapper" role="navigation">
```

Comment:
Has this nesting fuctionality been tested? cant view it with the current menu structure.

Reply:
No, but I'm not sure if we will have nesting on the menu going forward

Reply:
It was my understanding that nesting would not be needed with this new menu, since we have the additional vertical space. I can make nesting a thing, will discuss with @kakax114 as well.